packages/datadog-instrumentations/src/fastify.js

```diff
@@ -5,6 +5,7 @@ const { addHook, channel, AsyncResource } = require('./helpers/instrument')
 
 const errorChannel = channel('apm:fastify:middleware:error')
 const handleChannel = channel('apm:fastify:request:handle')
+const codeOriginForSpansChannel = channel('datadog:code-origin-for-spans')
```

Comment:
There shouldn't be explicit references to tracing in instrumentation.

Reply:
Also, channels are easier to reason about if they match the event that they represent. For example, different subscribers may be interested in different aspects of routing, so it would be better to have something like a routeChannel instead which would add the route when any subscriber is interested in that.

Reply:
I've pushed a commit with a fix. Let me know if that's what you had in mind